core/src/components/fab-button/readme.md

```diff
@@ -128,25 +128,25 @@ export default Example
-| `--background-activated` | Background of the button when activated |
-| `--background-focused`   | Background of the button when focused   |
-| `--border-color`         | Border color of the button              |
-| `--border-radius`        | Border radius of the button             |
-| `--border-style`         | Border style of the button              |
-| `--border-width`         | Border width of the button              |
-| `--box-shadow`           | Box shadow of the button                |
-| `--color`                | Text color of the button                |
-| `--color-activated`      | Text color of the button when activated |
-| `--color-focused`        | Text color of the button when focused   |
-| `--padding-bottom`       | Padding bottom of the button            |
-| `--padding-end`          | Padding end of the button               |
-| `--padding-start`        | Padding start of the button             |
-| `--padding-top`          | Padding top of the button               |
-| `--ripple-color`         | Color of the button ripple effect       |
-| `--transition`           | Transition of the button                |
+| Name                     | Description                                                                                               |
+| ------------------------ | --------------------------------------------------------------------------------------------------------- |
+| `--background`           | Background of the button                                                                                  |
+| `--background-activated` | Background of the button when activated                                                                   |
+| `--background-focused`   | Background of the button when focused                                                                     |
+| `--border-color`         | Border color of the button                                                                                |
+| `--border-radius`        | Border radius of the button                                                                               |
+| `--border-style`         | Border style of the button                                                                                |
+| `--border-width`         | Border width of the button                                                                                |
+| `--box-shadow`           | Box shadow of the button                                                                                  |
+| `--color`                | Text color of the button                                                                                  |
+| `--color-activated`      | Text color of the button when activated                                                                   |
+| `--color-focused`        | Text color of the button when focused                                                                     |
```

Comment:
More to add that could clarify Focused vs Activated? I actually don't know the difference myself....

Reply:
Done! https://github.com/ionic-team/ionic/pull/18538/commits/9a1cbb3fe5eb8279dfc9bbb4af4592199e06e4cf